spec/requests/health_check_spec.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+require "rails_helper"
+
+RSpec.describe "Health Check", type: :request do
+  describe "GET /health" do
+    it "has a health check" do
+      stub_aspace_login
+      get "/health.json"
+      expect(response).to be_successful
+    end
```

Comment:
Maybe add a test to these PRs that ensures / demonstrates it is not successful if something is down?